packages/telemetry/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@platformatic/telemetry",
+  "version": "0.31.0",
+  "description": "OpenTelemetry integration for Platformatic",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run lint | snazzy && tap test/*test.js",
+    "lint": "standard | snazzy"
+  },
+  "author": "Marco Piraccini <marco.piraccini@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/platformatic/platformatic.git"
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "snazzy": "^9.0.0",
+    "standard": "^17.1.0",
+    "tap": "^16.3.6"
+  },
+  "dependencies": {
+    "@opentelemetry/api": "^1.4.1",
+    "@opentelemetry/core": "^1.15.0",
+    "@opentelemetry/exporter-trace-otlp-proto": "^0.41.0",
+    "@opentelemetry/exporter-zipkin": "^1.15.0",
+    "@opentelemetry/resources": "^1.15.0",
+    "@opentelemetry/sdk-trace-base": "^1.15.0",
+    "@opentelemetry/semantic-conventions": "^1.15.0",
+    "fastify": "^4.20.0",
+    "fastify-plugin": "^4.5.0",
+    "install": "^0.13.0",
+    "npm": "^9.8.0"
```

Comment:
I don't think npm is needed as a dependency

Reply:
And `install` too :D